src/Command/AnalysisResult.php

```diff
@@ -38,6 +41,14 @@ public function __construct(
 		string $currentDirectory
 	)
 	{
+		usort(
+			$fileSpecificErrors,
+			function (Error $a, Error $b): int {
+				return strcmp($a->getFile(), $b->getFile())
+					?: ((int) $a->getLine() <=> (int) $a->getLine());
```

Comment:
Comparing by multiple arguments can be done with arrays, i.e 

~~~php
return [$a->getFile(), $a->getLine()] <=> [$b->getFile(), $b->getLine()];
~~~

Reply:
Interesting, didn't know that :+1: